pkg/proxy/ipvs/README.md

```diff
@@ -265,7 +265,7 @@ ip_vs
 ip_vs_rr
 ip_vs_wrr
 ip_vs_sh
-nf_conntrack_ipv4
+nf_conntrack_ipv4(replaced by nf_conntrack in kernel 4.19 and above)
```

Comment:
this info is just a few lines up 

> Ensure IPVS required kernel modules (**Notes**: use `nf_conntrack` instead of `nf_conntrack_ipv4` for Linux kernel 4.19 and later)